nixos/modules/virtualisation/containers.nix

```diff
@@ -368,6 +429,26 @@ in
               '';
             };
 
+            additionalCapabilities = mkOption {
+              type = types.listOf types.str;
+              default = [];
+              example = [ "CAP_NET_ADMIN" "CAP_MKNOD" ];
+              description = ''
+                Grant additional capabilities to the container.  See the
+                capabilities(7) and systemd-nspawn(1) man pages for more
+                information.
+              '';
+            };
+            enableTun = mkOption {
```

Comment:
I am unsure whether "enableTun" should be an explicit option or just a good example for "allowedDevices". It will probably be the primary use-case, but is it worth the special handling?

Reply:
For `/dev/fuse` you may find this info helpful: https://github.com/systemd/systemd/issues/2178#issuecomment-350168211